kview/src/main/java/dev/ikm/komet/kview/mvvm/view/genediting/SemanticFieldsController.java

```diff
@@ -187,7 +189,10 @@ private void initialize() {
         if (semantic != null && genEditingViewModel.getPropertyValue(MODE) == EDIT) {
                 //Change the button name to RESET FORM in EDIT MODE
                 clearOrResetFormButton.setText("RESET FORM");
+                submitMessage = "Semantic Details Edited Successfully!";
```

Comment:
Can we remove this assignment? Please see the comment below w.r.t. generating the toast message based on the MODE.